frontend/components/artists/artist_page_menu_bar.jsx

```diff
@@ -17,6 +15,18 @@ const ArtistPageMenuBar = ({
 		setArtistPageDropdownState({ isOpen: !artistPageDropdownState.isOpen });
 	};
 
+	// Create dropdown ref in parent component in order to wrap Redux container
+	const dropdownRef = useRef();
```

Comment:
What's the purpose of the `dropdownRef`?

Reply:
`dropdownRef` is used inside a [useEffect](https://github.com/imartinez921/versify_full-stack/pull/96/files#:~:text=const-,whenMenuIsOpen,-%3D%20(event) in the ArtistPageDropdown component so that React can tell whether a click is happening inside or outside the dropdown div in order to close it upon clicking outside.

I created the ref in the parent component because I plan for ArtistPageDropdown to be a recursive dropdown menu (just like `SongCardDropdown`) and I need all the submenus to be included in the same ref, not to create a ref each time.

Reply:
Ah got it, I was searching for `downdownRef` and not `ref`. Thanks for the explanation.

Reply:
No worries. Sorry for the variable switch. Wrapping a Redux container in a ref instead of a regular React component has extra steps, and since `ref` is a React keyword, I wanted to increase my chances of success lol.